plugins/versioning/src/main/kotlin/org/jetbrains/dokka/versioning/VersionsNavigationCreator.kt

```diff
@@ -68,9 +67,13 @@ class HtmlVersionsNavigationCreator(private val context: DokkaContext) : Version
                                     else
                                         versioningStorage.currentVersion.dir.resolve("not-found-version.html")
 
-                                a(href = absolutePath?.toRelativeString(position) +
-                                        if (!isExistsFile) "?v=" + version.urlEncoded() else "") {
-                                        text(version)
+                                val href =
+                                    absolutePath?.toRelativeString(position) + if (!isExistsFile) "?v=" + version.urlEncoded() else ""
+                                a(
+                                    href = href,
+                                    classes = if (!isExistsFile) "unavailable-version" else null
```

Comment:
I initially thought that the whole _version_ itself was unavailable, when in reality the _declaration_ for this version is unavailable.

So proposing to change it to `declaration-unavailable`. I also wonder if it's good UX to highlight such versions red, preventing the user from clicking it

Something like this, but prettier
![2022-11-26_07-31-05](https://user-images.githubusercontent.com/22685910/204075584-9afcc5e6-3409-4b71-ab62-c423a6e80627.png)

Reply:
>  I also wonder if it's good UX to highlight such versions red, preventing the user from clicking it

The style of top menu should not depend on a page.